src/Rings/mpoly-localizations.jl

```diff
@@ -2841,3 +2841,9 @@ end
 
 complement_of_point_ideal(m::MPolyIdeal) = complement_of_point_ideal(base_ring(m), rational_point_coordinates(m))
 
+function is_local(R::MPolyLocRing{<:Field})
+  return inverted_set(R) isa MPolyComplementOfPrimeIdeal || inverted_set(R) isa MPolyComplementOfKPointIdeal
```

Comment:
What about `MPolyPowersOfElement` and `MPolyProductOfMultSets`.

Reply:
They usually do not lead to local rings. Even if they did in special cases, there would be no generic check for this. But the user can still use `simplify` for this or even make sure that `is_local` and `is_known(is_local, ...)` return the correct answers.

Reply:
O.k., thx!

Reply:
I talked with @wdecker and we decided that we want a mathematically correct `is_local`. I have corrected the code once more in a place which had (for good reasons!) raised confusion.

Reply:
A mathematically correct is_local sounds good, if we also have the method which tests whether the value of is_local is known -- to catch errors before they are thrown.